rackcity/utils/change_planner_utils.py

```diff
@@ -143,7 +164,67 @@ def get_page_of_serialized_assets(all_assets, query_params):
         return all_assets[start:end]
 
 
-def get_many_assets_response_for_cp(request, change_plan):
+def get_many_assets_response_for_cp(request, change_plan, decommissioned=False):
```

Comment:
git is up to something. this entire method already existed. just added the `decommissioned=False` parameter. will comment on other lines that changed